force-app/main/default/classes/RD2_OpportunityEvaluation_BATCH.cls

```diff
@@ -201,11 +201,21 @@ public without sharing class RD2_OpportunityEvaluation_BATCH
     */
     @TestVisible
     private String getRDBatchQuery() {
+        Boolean rdBatchFilterOn = getOpportunityModifiedFilterEnabled();
+        String rdFilterWhereClause;
+        if (rdBatchFilterOn) {
+            Integer dateRange = getOpportunityModifiedDateRange();
+            Date earliestModifiedDate = currentDate.addDays(-dateRange);
+            rdFilterWhereClause = 'Id IN (SELECT npe03__Recurring_Donation__c' 
+                + 'FROM Opportunity WHERE LastModifiedDate > :earliestModifiedDate)';
```

Comment:
Hi @lparrott  - any chance we can switch this to use SystemModStamp instead of LastModifiedDate since SystemModStamp is indexed?